tools/coqdoc/output.ml

```diff
@@ -260,6 +260,40 @@ module Latex = struct
   let label_ident s =
     for i = 0 to String.length s - 1 do label_char s.[i] done
 
+
+  let convert_string f s =
+    let n = String.length s in
+    let rec qmap i =
+      if i == n then "" else (f s.[i]) ^ (qmap (i + 1))
+    in qmap 0
```

Comment:
I know that `coqdoc` is not critical performance-wise. But here that is pushing it a bit too far. Please use the `Buffer` module to build your string.

Reply:
Is it ok to use ocaml 4.07?  There is so much missing in 4.05, like String.to_seq and Seq.flat_map.

Reply:
Up to now no, we still declare 4.05 to be the minor version, but I'll raise the point this afternoon at the Coq Call.

Reply:
Honestly, `String.to_seq` is not worth the effort here. If you use `Buffer`, then you can use a plain `String.iter`.